### ISSUE TITLE
Add functions for showing alerts with less ceremony

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#48093]
Other platforms make showing a dialog that just shows a text string much easier. We could do the same by having functions such as the following in our framework:
```dart
Future<void> showMessage(BuildContext context, String message) {
  return showDialog(
    context: context,
    builder: (BuildContext context) => AlertDialog(
      content: Text(message),
    ),
  );
}
```

**Comment by HansMuller** [comment]
CC @shihaohong

**Comment by HansMuller** [comment]
We need to be careful that these functions don't become Swiss-army knives. Making it trivial to show a dialog with a string message would be helpful (so long as it's easy to discover the function). I think we should resist the inevitable temptation to add show-dialog functions with loads of parameters for action buttons and different kinds of messages.

**Comment by shihaohong** [comment]
I agree, I've noticed that with a lot of our more opinionated widgets, a lot of developers end up asking for more flexibility (for example, introducing a parameter that just wires through optional padding instead of having it hard-coded). 

We would have to be careful about this happening since devs might ask for some optional parameter and someone might simply make a new PR, then a function like `showMessage` starts to look a lot like `showDialog`.

**Comment by Hixie** [comment]
Yeah. In any case, I added this to Stretch Goals because I don't think it's something we should do in 2020.

**Comment by Hixie** [comment]
I wonder if this should just be a package...

**Comment by betterbets-privacy-policy** [comment]
The library Flutter GetX provide many ways to simplify things like Alert Dialogs, Navigation, Showing Snackbars, state management, etc.
